_posts/2020-08-26-bazel-3.5.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: posts
+title: "Bazel 3.5"
+authors:
+  - aiuto
+---
+
+[Bazel 3.5](https://github.com/bazelbuild/bazel/releases/tag/3.5.0) has just been released.
+
+Bazel 3.5 is a minor release. It is an incremental update to Bazel 3.4.
+
+Basline: 889bc0b523b47eeb38a72bf9bb6858ee525a7c7e
+
+Cherry picks:
+-   a7a0d48fbeb059ee60e77580e5d05baeefdd5699: Make no-op starlark transition not affect the output directory.
+-   b37c51c7085f0aefe04034dd451acb847605ddb5: Add include_prefix and strip_include_prefix to cc_common.compile
+-   f6ad35fcde93f92c591778ed7db38d167f5bbc03: Delete --experimental_transparent_compression
+-   39bc97eab295bddb35b38bfc4a2ff3d2b15d034e: Remove --experimental_action_args
+-   b9706675a7abf6ceebb250f0b3dfa4087a0c35f6: Stop needlessly parsing WORKSPACE files from external repositories.
+-   e574d558da17cfd0f818e7a937a07926aa270069: Allow hyphen char in workspace name
+-   9993785fa0c4fa4172aa31d306f3abea76833abf: Allow dot ('.') in workspace names.
+
+## Incompatible changes
+
+- Removed the `--experimental_process_wrapper_wait_fix` flag.
+- Removed the `--experimental_ui_deduplicate` flag.
+- Removed the `--experimental_transparent_compression` flag.
+- Removed the `--experimental_action_args` flag.
+- Bazel now correctly prefers Xcode versions in `/Applications` over
+  any other paths, which resolves an issue with accidentally picking up
+  an Xcode version from a Time Machine backup or network disk. In the
+  improbable case that you relied on the old behavior and Bazel now picks
+  up Xcode from the wrong location, you can fix it by moving that Xcode
+  version to /Applications.
+
+## New features:
+
+- `cquery` now follows aspects when the `--include_aspects` flag is used.
+- Add support to bazel/crosstool for building arm64 on macos.
+
+## Important changes:
+
+- Fixed resource shrinking when \<overlayable/> tags are used.
+- Remove legacy handling of `--extra_checks`
```

Comment:
I don't know what this means or where I can read more about it. Is it an incompatible change?

Reply:
It appears to be another RELNOTES for something that should be a non-op. Removed.